src/events/message.js

```diff
@@ -34,67 +34,101 @@ function throttleUser(
   callback,
   throttleTime = config('bot.msgThrottleTime'),
 ) {
-  if (retries.has(author.username)) {
+  if (retries.has(author.id)) {
     handleRetry(author);
-    return;
+    throw Error('Preventing flood.');
   }
 
   setInterval(() => {
-    retries.delete(author.username);
+    retries.delete(author.id);
   }, throttleTime * 1000);
 
-  retries.set(author.username, 1);
+  retries.set(author.id, 1);
 
   callback();
 }
 
 module.exports = async message => {
-  if (message.author.bot) {
+  const { author, client, content, channel } = message;
+
+  if (author.bot) {
     return;
   }
 
-  const botUsername = message.client.user.toString();
+  const botUsername = client.user.toString();
 
-  if (message.content.startsWith(botUsername)) {
-    const [username, tag, ...gibberish] = message.content.trim().split(' ');
+  if (content.startsWith(botUsername)) {
+    const allowedTags = [...getTags().keys()].filter(
+      key => key !== '__reserved',
+    );
+
+    const [, tag, ...gibberish] = content.trim().split(' ');
 
     if (gibberish.length > 0) {
       const attachment = new Attachment(getReservedImage('angryPepe'));
 
-      message.channel.send('**EI _BORDA_, UMA TAG DE CADA VEZ, OK?!?!**', {
-        files: [attachment],
-      });
+      try {
+        const warned = await author.send(
+          '**EI _BORDA_, UMA TAG DE CADA VEZ, OK?!?!**',
+          { files: [attachment] },
+        );
 
-      return;
-    }
+        if (warned) {
+          await author.send(`Tags :: ${allowedTags.join(' , ')}`);
```

Comment:
What about `', '` instead? Or does it look worse when displaying?

Reply:
Don't know... Did it this way to make less error prone when the user copy/paste it.

@musemartins mind to untie this?

Reply:
Will leave it as it is.